src/Config/Config.php

```diff
@@ -69,7 +69,7 @@ class Config
         'klarnapaylater' => ['eur'],
         'klarnasliceit' => ['eur'],
         'mybank' => ['eur'],
-
```

Comment:
Could you constantas more, voucher could have it's own constanta which could be reused instead of 'voucher' where relevant. Also in this case if klarnapaylater = PaymentMethod::KLARNA_PAY_LATER, then klarnapaylater could be replaced by constanta.

Reply:
Its on todo list, also vouncher is on beta so waiting for full api update so we could use static name.